public-ui/src/app/data-browser/views/ehr-view/components/concept-row-react.component.tsx

```diff
@@ -0,0 +1,388 @@
+               <div id={id}>
+                 <div className='tbl-exp-r' onClick={() => this.expandRow()}>
+                 <div className={tblClass} style={domain === 'labs & measurements' ? styles.tblRLabs : styles.tblRow}>
+                 <div className='body-lead tbl-d' style={styles.bodyLead}>
+                    <span>{conceptIndex}. </span>
+                    <HighlightReactComponent searchTerm={searchTerm} text={concept.conceptName}/>
+                 </div>
+                 <div className='body-lead tbl-d' style={styles.bodyLead}>
+                    {concept.countValue <= 20 && <span>&le; </span>} {concept.countValue.toLocaleString()}
+                 </div>
+                 <div className='body-lead tbl-d' style={styles.bodyLead}>
+                    {this.participantPercentage(concept.countValue)} %
+                 </div>
+                 {domain === 'labs & measurements' && concept.measurementConceptInfo &&
+                 <div className='body-lead tbl-d'>
+                    {concept.measurementConceptInfo.hasValues === 1 ?
+                    <span className='test-span' style={styles.measurementTypeSpan}><i className='fas fa-vial' style={{'transform': 'rotate(315deg)'}}></i>
+                    <TooltipReactComponent tooltipKey='valueFilter' label='' searchTerm='' action='' /></span> :
+                    <span className='order-span' style={styles.measurementTypeSpan}><i className='far fa-file-signature'></i>
+                   <TooltipReactComponent tooltipKey='orderFilter' label='' searchTerm='' action='' /></span>}
+                 </div>
+                 }
+                 <div className='body-lead tbl-d icon-btn-group' style={styles.iconBtnGroup}>
+                    <button className='icon-btn' onClick={(e) => {e.stopPropagation(); this.showChart('non-sources'); }}>
+                        <ClrIcon shape='bar-chart' className={(showConceptChart && graphToShow !== GraphType.Sources) ?
+                        'is-solid icon-choice' : 'icon-choice'} style={{width: 20, height: 20, color: '#2691D0'}}/>
+                    </button>
+                    <button className='icon-btn icon-choice' onClick={(e) => {e.stopPropagation(); this.showChart('sources'); }}>
+                        <div className={(showConceptChart && graphToShow === GraphType.Sources) ? 'source-btn-active' : 'source-btn'}
+                        style={styles.sourceBtnMeta} >
+                        </div>
+                    </button>
+                    <button className='icon-btn'>
+                         <ClrIcon shape='share' className='icon-choice' style={{width: 20, height: 20, color: '#2691D0'}}
+                         onClick={(e) => {e.stopPropagation(); this.shareConcept(e); }}>
+                         </ClrIcon>
+                    </button>
+                    {showCopyAlert &&
+                        <div style={{position: 'absolute', top: alertBoxTopStr, left: alertBoxLeftStr}}>
```

Comment:
We should be able to do this with static styles. This seemed to get basically the same position, if you could verify:
```
style={{margin: '20px -60px 0 0', position: 'absolute'}}
```
That would allow us to remove `alertBoxTopStr` and `alertBoxLeftStr` from state and cleanup `shareConcept()` and `render()` a little also.

Reply:
Thanks. Cleaned things up.